### ISSUE TITLE
update graph saving rec?

### ISSUE THREAD

**Issue by maelle** [igraph/rigraph#1242]
> You can also just save the R workspace and restore it later.

in the intro vignette. It does not sound optimal to me, and could use actual example code?

**Comment by maelle** [comment]
@schochastics what would you recommend to users to save and restore graphs? I can then update the vignette (and its translation).

**Comment by schochastics** [comment]
yes we should not advocate saving the workspace. I usually advise to use `saveRDS()` or equivalent formats. But i think for the "generic" R user `saveRDS()` should be fine

**Comment by maelle** [comment]
so not `write_graph()`?

**Comment by schochastics** [comment]
I only ever used `read_graph()` and `write_graph()` to be able to use the network in the respective software tool. I think if it is foreseeable that the network is only used in R with igraph, then `saveRDS` is safer. If a nonbinary format is needed, then I usually advocate to write to two data frames

```r
g <- make_graph("Zachary")

#write
graph_df <- as_data_frame(g,"both")
write.csv(graph_df$vertices,"vertices.csv")
write.csv(graph_df$edges,"edges.csv")

#read
edges <- read.csv("edges.csv")
vertices <- read.csv("vertices.csv")
graph_from_data_frame(edges,vertices = vertices)
```
Might be more cumbersome but more in line with tidy principles and easier to understand

**Comment by szhorvat** [comment]
> I only ever used `read_graph()` and `write_graph()` to be able to use the network in the respective software tool.

Long-term archival is also a good reason. Will igraph 3.0 be able to use the data structure of igraph 2.0? So far we tried hard to maintain compatibility, but it's a concern.

**Comment by maelle** [comment]
in the vignette, since it's for beginners, let's go with RDS for simplicity.
